modules/rostests/apitests/shell32/SHGetAttributesFromDataObject.cpp

```diff
@@ -13,10 +13,11 @@
 
 
 static CLIPFORMAT g_DataObjectAttributes = 0;
-static const DWORD dwDefaultAttributeMask = SFGAO_CANCOPY | SFGAO_CANMOVE | SFGAO_STORAGE | SFGAO_CANRENAME | SFGAO_CANDELETE |
-                                     SFGAO_READONLY | SFGAO_STREAM | SFGAO_FOLDER;
+static const DWORD dwDefaultAttributeMask = SFGAO_CANCOPY | SFGAO_CANMOVE | SFGAO_STORAGE | SFGAO_CANRENAME |
+                                            SFGAO_CANDELETE | SFGAO_READONLY | SFGAO_STREAM | SFGAO_FOLDER;
 static_assert(dwDefaultAttributeMask == 0x2044003B, "Unexpected default attribute mask");
-
+static const DWORD dwDefaultAttributeMask_WS03 = dwDefaultAttributeMask | SFGAO_FILESYSTEM | SFGAO_CAPABILITYMASK;
+static_assert(dwDefaultAttributeMask_WS03 == 0x6044017F, "Unexpected default attribute mask for WS03, Vista");
```

Comment:
The var name doesn't convey the fact it holds for Vista as well.
Something like `dwDefaultAttributeMask_WS03_Vista` ? or `dwDefAttribMask_WS03_Vista` (with similar name for the other) if we want slightly shorter names?